packages/xo-web/src/xo-app/hub/recipes/recipe-ev.js

```diff
@@ -0,0 +1,108 @@
+import _ from 'intl'
+import ActionButton from 'action-button'
+import ButtonLink from 'button-link'
+import decorate from 'apply-decorators'
+import Icon from 'icon'
+import marked from 'marked'
+import React from 'react'
+import { Card, CardBlock, CardHeader } from 'card'
+import { form } from 'modal'
+import { connectStore } from 'utils'
+import { createGetObjectsOfType } from 'selectors'
+import { createEasyVirtVM } from 'xo'
+import { injectState, provideState } from 'reaclette'
+import { success } from 'notification'
+import { withRouter } from 'react-router'
+
+import RecipeForm from './recipe-form-ev'
+
+const RECIPE_INFO = {
+  id: 'test',
+  name: 'DCScope VM',
+  description: 'Creates an DCScope VM with parameters and application inside',
+}
+
+export default decorate([
+  withRouter,
+  connectStore(() => ({
+    pools: createGetObjectsOfType('pool'),
+    recipeCreatingResources: state => state.recipeCreatingResources,
+  })),
+  provideState({
+    initialState: () => ({
+      selectedInstallPools: [],
+    }),
+    effects: {
+      async create() {
+        const { markRecipeAsCreating, markRecipeAsDone } = this.props
+        const recipeParams = await form({
+          defaultValue: {
+            pool: {},
+          },
+          render: props => <RecipeForm {...props} value={{ nbNodes: 1, ...props.value }} />,
```

Comment:
maybe `parentState` is more easy to understand?
```
          render: props => <RecipeForm {...props} parentState={{ nbNodes: 1, ...props.value }} />,
```

Reply:
Do I need to change every mention of value in the child state after this change?